File: app/src/main/java/com/chirag/covid19india/webservice/APIs.java
package com.chirag.covid19india.webservice;

import static com.chirag.covid19india.util.AppConstants.BASE_DOMAIN;

public class APIs {


    static final String BASE_URL = BASE_DOMAIN;
    public static final String BASE_IMAGE_PATH = BASE_DOMAIN;

    public static final String LOAD_MORE_LIMIT = "10";

    public static final String DEMO_API = "https://api.rootnet.in/covid19-in/unofficial/covid19india.org/statewise";


    //National Data
    public static final String API_NATIONAL_DATA = "/data.json";

    //State-District Data
    public static final String API_STATE_DISTRICT_DATA = "/v2/state_district_wise.json";

    //Daily Updates
    public static final String API_DAILY_UPDATES = "updatelog/log.json";

}

File: app/src/main/java/com/chirag/covid19india/webservice/APIs.kt
package com.chirag.covid19india.webservice

import com.chirag.covid19india.util.AppConstants

object APIs {
    const val BASE_URL = AppConstants.BASE_DOMAIN

    //National Data
    const val API_NATIONAL_DATA = "/data.json"

    //State-District Data
    const val API_STATE_DISTRICT_DATA = "/v2/state_district_wise.json"

    //Daily Updates
    const val API_DAILY_UPDATES = "updatelog/log.json"
}